src/languages/php.js

```diff
@@ -102,11 +103,50 @@ export default function(hljs) {
     // <https://www.php.net/manual/en/reserved.classes.php>
     'Directory __PHP_Incomplete_Class parent php_user_filter self static stdClass'
   };
+  const ATTRIBUTES = {
+    className: 'meta',
```

Comment:
@joshgoebel 

Answers:
1. There will be. They are better than annotations, replace them and annotations are just code in comments. Attributes are part of the language.
2. I made complex cases, to test the code, but yes.   [Api Platform](https://api-platform.com/docs/core/graphql/#filters) , [Doctrine](https://www.doctrine-project.org/2021/05/24/orm2.9.html)
3. PhpStorm is one of the most popular and highlights them like [this](https://blog.jetbrains.com/phpstorm/2020/10/phpstorm-2020-3-eap-4/) and [this](https://github.com/JetBrains/phpstorm-attributes#deprecated). Opening `#[`, closing `]` and the name of the attribute is meta
4. When I was working on it, I checked how Highlight js presents attributes in other languages, for example: [C#](https://docs.microsoft.com/pl-pl/dotnet/csharp/programming-guide/concepts/attributes/creating-custom-attributes), [Java](https://en.wikipedia.org/wiki/Java_annotation) and [Rust](https://doc.rust-lang.org/reference/attributes.html). I guess they don't have such complex nested cases, so probably I used too much meta tag.
In [this blog](https://stitcher.io/blog/attributes-in-php-8#rundown) the name of the attribute is also a 'title'.
In my opinion, the class names should be 'title' like in PhpStorm and the name of the attribute should be meta. 
I have no idea about the words like `allowMissingFields`.  Is any other language use some other classes than: keywords, title, meta ...?

Reply:
> PhpStorm is one of the most popular and highlights them like this and this. Opening #[, closing ] and the name of the attribute is meta

I think this would be a good thing to model, and should be doable.

> In this blog the name of the attribute is also a 'title'.

Looks like `type` to be me which surely is not correct.

> I have no idea about the words like allowMissingFields

I would assume code inside attributes would fall back to using the same highlighting as outside code... so perhaps this would be highlighted as some type of hash attribute, or not at all.

Reply:
@joshgoebel Ok so I will go with the PhpStorm highlighting.

About `allowMissingFields` this could be the same as [named arguments](https://stitcher.io/blog/new-in-php-8#named-arguments-rfc) (which is also not implemented in highlight js)

I will pick [something](https://github.com/highlightjs/highlight.js/blob/main/src/styles/a11y-dark.css).

Reply:
We do have `attribute` and `attr` for such things, see docs.